packages/ui/src/components/smui-wrapper/custom-drawer/custom-drawer.store.ts

```diff
@@ -0,0 +1,61 @@
+// SVELTE
+import { writable, get } from 'svelte/store'
+// TYPES
+import type { DrawerPayload } from './custom-drawer.types'
+
+//==== STATE
+const open = writable<boolean>(false)
```

Comment:
We can simplify the use of `open`, `currentTitle`, `currentDescription`, `currentInnerComponent`, and `currentInnerComponentProps` by combining them all together into a single object.

Reply:
Refactored see 584f1fc